### ISSUE TITLE
Add a layout editor for the touch controller

### ISSUE THREAD

**Issue by Ashwinnnndt** [doukutsu-rs/doukutsu-rs#254]
The touchscreen controller layout should be customisable and the weapon switch without opening menu should be implemented. That is switching weapons left to right should be there. In freeware, it exists

**Comment by Veha0001** [comment]
![IMG_20240309_072926_123](https://github.com/doukutsu-rs/doukutsu-rs/assets/84841152/4cf7d62d-1582-4ab5-8763-52a4b638bbf5)

**Comment by SethPeters91** [comment]
Yeah, the lack of on-the-fly weapon switching is killing this port for me. I appreciate the immense amount of effort that probably went into this project but I am perplexed as to how anyone actually was able to play all the way till the end in the state that it's in. It's impressive that it's able to run and have all the features that it has, but if you can't even control what you're doing half of the time it ends up feeling more like a novelty rather than an actual experience worth playing.

**Comment by alula** [comment]
Give suggestions how to improve it instead of complaints please.

**Comment by SethPeters91** [comment]
> Give suggestions how to improve it instead of complaints please.

🤨 Okay...

> The touchscreen controller layout should be customisable and the weapon switch without opening menu should be implemented. That is switching weapons left to right should be there. In freeware, it exists

Literally all I could ask for besides controller support.
